tests/unit_test/action/kremote_action_test.py

```
@@ -0,0 +1,103 @@
+import pytest
+from aioresponses import aioresponses
+from yarl import URL
```

Ruff

`yarl.URL` imported but unused

Remove unused import: `yarl.URL`

(F401)